beacon_node/eth2_libp2p/src/rpc/methods.rs

```diff
@@ -141,6 +141,9 @@ pub enum GoodbyeReason {
     /// The peer is banned
     Banned = 251,
 
+    /// The IP address the peer is using is banned.
+    BannedIP = 252,
```

Comment:
Is this value specced somewhere?

Reply:
Nah. These usually come with an error message. Other clients make these up as well as us and after seeing them we update. Usually we just print the error message for unknown ids. 

In the specs we were a bit lazy and said: 
> Clients MAY use reason codes above 128 to indicate alternative, erroneous request-specific responses.

Reply:
We the 4 there are also not specced but we came to consensus with other clients.

Reply:
Ok cool. I guess we can tell them about this value in good time